docs/pages/server-access/guides/openssh.mdx

```diff
@@ -38,20 +38,10 @@ certificates generated by the Teleport Auth Service.
 
 Start by exporting the Teleport CA public key.
 
-On your local machine, print the Teleport certificate authority certificate to
-stdout:
-
-```code
-$ tctl auth export --type=user | sed "s/cert-authority\ //"
-```
-
-Copy the output. 
-
 On the host where you are running `sshd`, run the following commands.
 
-Assign the output of the `tctl auth export` command to an environment variable:
-
 ```code
+$ curl 'https://teleport.example.com/webapi/auth/export?type=user' | sed "s/cert-authority\ //"
 $ export KEY="<pasted output>"
```

Comment:
`pasted output` might be a bit misleading now. Maybe use shell command substitution to assign `KEY` to the output of the `curl` command?

Reply:
Yes :+1:

Reply:
Just tested and it works:

```
dinis@lenix ~> export KEY=(curl -k -f 'https://127.0.0.1.nip.io:3080/webapi/auth/export?type=user' | sed "s/cert-authority\ //"); echo $KEY
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   424  100   424    0     0   6580      0 --:--:-- --:--:-- --:--:--  6625
ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC1BQPPLArth3gx40Cy4KNEHFYFpu0Cqu1hL1zMhu2Qgcok4V3VJJKxuMffNZN3XcBj87+6RhFgKruNsZEXzfuzmSl5ZZlLuldsVs3QbZ9Wf/foOKyUfjzJ+SRqKVlbkxVWsc1GZZLP9cq/BZukBRBvWaDhdC9OS/LQD2fzBUFJVFxJaT4nY/blu+1DCoDFwI9pc8YTaUaZIXRe8Vu6vWq3xoxeC8hWLavRXy5hOzu8+07hal0qAAqKZohvckM7xOnYu5w3wJHP3zmj6HLAI9eyYoZ6AkBikklDQLFe4R8UOTDFMnpThJSO2DHXiVPIy3Xms209gF+1evYxiRVSomIz clustername=lenix&type=user
```